package.json

```diff
@@ -2,9 +2,14 @@
   "name": "platform-chaos",
   "version": "1.0.0",
   "description": "A node sdk for building services capable of injecting chaos into PaaS offerings",
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "test": "mocha"
+    "pretest": "npm run cleanup && npm run compile",
+    "test": "npm run test-only",
+    "posttest": "npm run cleanup",
+    "test-only": "mocha dist/test/",
+    "compile": "npx tsc -p tsconfig.json",
+    "cleanup": "rm -rf dist/"
```

Comment:
are there any docs for travis that support either a need for this or a suggestion? ie will travis run these builds in containers that get cleaned up?